lib/make-disk-image.nix

```diff
@@ -140,7 +154,13 @@ in
       done
       ${installer}
     ''}
-    export QEMU_OPTS=${lib.escapeShellArg "${QEMU_OPTS} -m 1024"}
+
+    if [[ -z $build_memory ]]; then
+      build_memory=1024
+    fi
```

Comment:
```suggestion
    build_memory=''${build_memory:-1024}
```

Reply:
If I apply that locally the script fails because the var is just empty and qemu complaints about a missing value, but I don't know why since the syntax you suggested works fine if I do it in a test script.

Reply:
any idea why that is?

Reply:
fixed